apps/happychat/src/index.ejs

```diff
@@ -23,6 +23,7 @@
 		<div id="wpcom" class="container"></div>
 		<script>
 			window.configData = {
+				env_id: 'help-center',
```

Comment:
Turns out Happychat can die suddenly without this variable. And honestly I didn't investigate why.